### PR TITLE
*/*: Homepage updates

### PR DESCRIPTION
**Pull request by mm1ke**
Hi,

This simply updates a few `HOMEPAGES` of various ebuilds.

### CODE REVIEW

File: sys-cluster/zetcd/zetcd-0.0.4-r2.ebuild [hunk 1]
```diff
@@ -17,8 +17,8 @@ inherit golang-build golang-vcs-snapshot
 
 ARCHIVE_URI="https://${EGO_PN}/archive/v${PV}.tar.gz -> ${P}.tar.gz"
 
-DESCRIPTION="A ZooKeeper personality for etcd"
-HOMEPAGE="https://github.com/coreos/zetcd"
+DESCRIPTION="ZooKeeper personality for etcd"
+HOMEPAGE="https://github.com/etcd-io/zetcd"
```
Comment (AndrewAmmerlaan) [inline sys-cluster/zetcd/zetcd-0.0.4-r2.ebuild:21]: `EGO_PN=` a couple of lines above also needs updating

File: sys-devel/parity/parity-9999.ebuild [hunk 2]
```diff
@@ -8,11 +8,11 @@ if [[ ${PV} == 9999 ]]; then
 	EGIT_REPO_URI="git@github.com:ssi-schaefer/parity.git https://github.com/ssi-schaefer/parity.git"
 	BDEPEND="dev-util/confix"
 else
-	SRC_URI="https://github.com/mduft/${PN}/releases/download/${PV}/${P}.tar.bz2"
+	SRC_URI="https://github.com/ssi-schaefer/${PN}/releases/download/${PV}/${P}.tar.bz2"
```
Comment (AndrewAmmerlaan) [inline sys-devel/parity/parity-9999.ebuild:11]: Let's just drop the 9999 ebuild here, it depends on `dev-util/confix` which does not exist.

**Review by AndrewAmmerlaan — COMMENTED** [review]
Excellent work here!

Here's the full list of complaints from pkgcheck (don't worry, no need to fix them all), there are still some `RedirectedUrl` warnings. Might I ask you to also update the url's in the relevant `metadata.xml` files where applicable?
```
dev-ml/easy-format
RedirectedUrl: version 1.2.0: metadata.xml: remote-id: permanently redirected: https://github.com/mjambon/easy-format -> https://github.com/ocaml-community/easy-format

games-arcade/savagewheels
UnstableOnly: for arch: [ amd64 ], all versions are unstable: [ 1.6.1-r2 ]
RedirectedUrl: version 1.6.1-r2: metadata.xml: remote-id: permanently redirected: https://github.com/petarov/savagewheels -> https://github.com/kenamick/savagewheels

dev-ml/easy-format
UnstableOnly: for arch: [ amd64 ], all versions are unstable: [ 1.2.0 ]

sys-cluster/zetcd
UnstableOnly: for arch: [ amd64 ], all versions are unstable: [ 0.0.4-r2 ]
DeprecatedEclass: version 0.0.4-r2: uses deprecated eclass: golang-build (migrate to go-module.eclass)
DeprecatedEclass: version 0.0.4-r2: uses deprecated eclass: golang-vcs-snapshot (migrate to go-module.eclass)
RedirectedUrl: version 0.0.4-r2: metadata.xml: remote-id: permanently redirected: https://github.com/coreos/zetcd -> https://github.com/etcd-io/zetcd
RedirectedUrl: version 0.0.4-r2: SRC_URI: permanently redirected: https://github.com/google/go-genproto/archive/ee236bd376b077c7a89f260c026c4735b195e459.tar.gz -> https://github.com/googleapis/go-genproto/archive/ee236bd376b077c7a89f260c026c4735b195e459.tar.gz

net-irc/irssi-xmpp
UnstableOnly: for arches: [ amd64, x86 ], all versions are unstable: [ 0.54, 0.54-r1 ]
RedundantVersion: version 0.54: slot(0) keywords are overshadowed by version: 0.54-r1

sys-cluster/zetcd
RedirectedUrl: version 0.0.4-r2: SRC_URI: permanently redirected: https://github.com/coreos/etcd/archive/714e7ec8db7f8398880197be10771fe89c480ee5.tar.gz -> https://github.com/etcd-io/etcd/archive/714e7ec8db7f8398880197be10771fe89c480ee5.tar.gz

net-libs/net6
UnstableOnly: for arches: [ arm, hppa ], all versions are unstable: [ 1.3.14-r2 ]
PotentialStable: version 1.3.14-r2: slot(0), stabled arches: [ amd64, ppc, x86 ], potentials: [ ~arm, ~hppa ]

sys-devel/parity
UnderscoreInUseFlag: USE flag 'vc10_0' uses reserved underscore character
UnderscoreInUseFlag: USE flag 'vc11_0' uses reserved underscore character
UnderscoreInUseFlag: USE flag 'vc12_0' uses reserved underscore character
UnderscoreInUseFlag: USE flag 'vc14_0' uses reserved underscore character
UnderscoreInUseFlag: USE flag 'vc8_0' uses reserved underscore character
UnderscoreInUseFlag: USE flag 'vc9_0' uses reserved underscore character
NonexistentDeps: version 9999: BDEPEND: nonexistent package: dev-util/confix

sys-cluster/zetcd
RedirectedUrl: version 0.0.4-r2: SRC_URI: permanently redirected: https://github.com/coreos/zetcd/archive/v0.0.4.tar.gz -> https://github.com/etcd-io/zetcd/archive/v0.0.4.tar.gz

dev-python/mamba
PythonGHDistfileSuffix: version 0.11.2: GitHub archive 'mamba-0.11.2.tar.gz' ('https://github.com/nestorsalceda/mamba/archive/v0.11.2.tar.gz') is not using '.gh.tar.gz' suffix

net-irc/irssi-xmpp
RedirectedUrl: version 0.54-r1: HOMEPAGE: permanently redirected: https://cybione.org/~irssi-xmpp/ -> https://github.com/cdidier/irssi-xmpp
```

**Comment by mm1ke** [comment]
@AndrewAmmerlaan 
Thanks for the review. I've updated the commits and hopefully i haven't forgot anything.
I hope it's fine that i haven't made extra commits for the `metadata.xml` fixup's?

**Pull request closed without merging**